ui/app/hooks/useUserPreferencedCurrency.js

```diff
@@ -0,0 +1,52 @@
+import { preferencesSelector, getShouldShowFiat } from '../selectors'
+import { useSelector } from 'react-redux'
+import { PRIMARY, SECONDARY, ETH } from '../helpers/constants/common'
+
+/**
+ * Defines the shape of the options parameter for useUserPreferencedCurrency
+ * @typedef {Object} UseUserPreferencedCurrencyOptions
+ * @property {number} [fiatNumberOfDecimals] - Number of significant decimals to display
```

Comment:
Looks like a typo was made here:
```suggestion
 * @property {number} [numberOfDecimals] - Number of significant decimals to display
```